### PR TITLE
<feat>: add function np.any() and np.where().Also with some UnitTest

### PR DESCRIPTION
**Pull request by zhuoshui-AI**
<feat> add function np.any() and np.where() in logits and APIs.
Also with some UnitTest

### CODE REVIEW

**Comment by Nucs** [comment]
Thank you a lot for your contribution,
These np methods were rewritten in the recent preview, thank you a lot!

**Comment by zhuoshui-AI** [comment]
你好，邮件已收到
